app/models/fee.rb

```diff
@@ -100,6 +100,23 @@ def group_name
     charge&.group_properties&.find_by(group:)&.invoice_display_name || group&.name
   end
 
+  def grouped_by_display
```

Comment:
can we use helpers in views instead of put this in the model? 🙏

Reply:
fixed